src/shared/vendor/third_party/webcomponents.js

```diff
@@ -6374,4 +6374,6 @@ CustomElements.addModule(function(scope) {
   window.Platform = scope;
 })(window.WebComponents);
 
-module.exports = window.WebComponents;
+if (typeof exports !== 'undefined') {
```

Comment:
I think the intention is probably to crash if `exports` exists but `module` doesn't because the wrapper doesn't know how to deal with that. I'll leave as-is because I think either should work fine and this seems standard.